Makefile

```diff
@@ -40,6 +40,9 @@ agent_tests:
 agent_add_node:
 	./agent/07_agent_add_node.sh
 
+agent_remove_node:
```

Comment:
Ona second check, it seems more suitable naming the targets as:

* `agent_add_extraworker_nodes`
* `agent_remove_extraworker_nodes`

Reply:
Good suggestion. I will make that change.